.github/workflows/build.yml

```diff
@@ -239,7 +239,7 @@ jobs:
               with:
                   context: .
                   platforms: ${{ matrix.platform }}
-                  push: ${{ github.ref == 'refs/heads/main' || (github.event_name == 'workflow_dispatch' && github.event.inputs['build-images']) }}
+                  push: true #${{ github.ref == 'refs/heads/main' || (github.event_name == 'workflow_dispatch' && github.event.inputs['build-images']) }}
```

Comment:
We probably need to remove the source distribution.

What about windows then ?